csm/core/services/s3/server_info.py

```diff
@@ -15,18 +15,26 @@
 
 from csm.common.services import ApplicationService
 from csm.common.service_urls import ServiceUrls
+from csm.common.errors import InvalidRequest
 
 
 class S3ServerInfoService(ApplicationService):
     """
     Service for acessing S3 server information
     """
 
-    async def get_s3_server_info(self):
+    async def get_s3_server_info(self, schemas):
         """
         Obtain information about S3 server in json format
```

Comment:
We forgot to add documentation to the new `schemas` parameter here.  Also, let's add documentation for the return value as well.

Reply:
@tb-s Added documentation. please check.

Reply:
I believe you added documentation to another function.  Also, I have just noticed that this function does not need to be `async` anymore.  We can remove it and adapt its calls.